src/main/java/com/WebController/EventController.java

```diff
@@ -1,15 +1,22 @@
 package com.WebController;
 
-import com.Model.Patient;
-import com.Model.PatientEventAssociation;
-import com.Repository.AssociationRepository;
-import com.Repository.EventRepository;
-import com.Repository.PatientRepository;
-import com.Model.Event;
+import com.Model.*;
+import com.Repository.*;
+import com.Service.EmailSenderService;
+import com.Service.MessageSender;
+import org.assertj.core.util.IterableUtil;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
 
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
 import java.util.List;
 
```

Comment:
maybe just make this a @RestController so you don't need to use @ResponseBody in all your methods. This forces you to think in terms of a REST API here.